outrank/core_ranking.py

```diff
@@ -130,9 +130,13 @@ def mixed_rank_graph(
     # Map the scoring calls to the worker pool
     pbar.set_description('Allocating thread pool')
 
+    reference_model_features = {}
+    if 'prior' in args.heuristic:
```

Comment:
you check for `-prior` at some point, but `prior` at some other point. Consider creating a helper function `is_prior_heuristic` or something, that unifies this behavior (and centralizes it)